src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo_link {
+  text-decoration: none;
+  display: flex;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+  margin: 10px auto;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  display: flex;
+  margin-left: 20px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both to avoid margin collapsing issues.